components/CategoriesList.tsx

```
@@ -21,33 +28,110 @@ const Component = ({
   selectedCategory,
   preventClose,
   selectedCategories,
+  swipeable,
 }: CategoriesListProps) => {
   const { bottom } = useSafeAreaInsets();
   const { close } = useBottomSheet();
+
+  const onEdit = (category: CategoryModel) => {
+    // Implement edit logic here
+    console.log({ category });
```

Biome (1.9.4)

[error] Don't use console.log

console.log is usually a tool for debugging and you don't want to have that in production.
If it is not for debugging purpose then using console.info might be more appropriate.
Unsafe fix: Remove console.log


(lint/suspicious/noConsoleLog)